tests/unit/test_reconcile.py

```diff
@@ -1035,3 +1036,116 @@ def test_index_page(
     returned_action = reconcile.index_page(index=index, table_rows=table_rows)
 
     assert returned_action == expected_action
+
+
+@pytest.mark.parametrize(
+    "actions, expected_value",
+    [
+        pytest.param(
+            [types_.NoopAction(1, tuple("path"), types_.Navlink("title", None, True), None)],
```

Comment:
should the factories be used for these instead?

Reply:
Definitely. I did not see the factories for this! Thanks!